lib/cell.rb

```diff
@@ -24,20 +24,20 @@ def fired_upon?
   def fire_upon
     if @fired_upon == false
       @fired_upon = true
-      ship.hit if empty? == false
+      ship.hit unless empty?
```

Comment:
Really like the refactoring on this line. Cleaner and simpler. Nice use of "unless".